_examples/01_one_account_one_region/README.md

```diff
@@ -0,0 +1,155 @@
+# WIP
+
+This example demonstrates several basic Terragrunt best practices by
+creating a simple infrastructure with two environments (production and staging),
+which is hosted in a single AWS account in a single AWS region.
+
+**Demonstrated concepts and keywords:**
+
+* Recommended Terragrunt directory structure
+* Basic Terragrunt configuration in `terraform.tfvars` files
+* Keeping Terragrunt configurations DRY by using the `include` keyword and 
+  the `find_in_parent_folders()` Terragrunt built-in function
+* Using remotely sourced and independently versioned Terraform modules
+* Independent remote state for each individual infrastructure component
+* Defining dependencies between infrastructure components using the Terragrunt 
+  `dependencies` keyword
+* Sharing remote state between infrastructure components using `terraform_remote_state`
+
+## Running the example
+To create the staging environment:
+```
+$> cd live/staging
+$> terragrunt apply-all
+```
+
+
+To create the production environment:
+```
+$> cd live/production
+$> terragrunt apply-all
+```
+
+## Directory structure
+Following best practices, the following directory structure contains all of our Terragrunt
+configuration:
+
+```
+live                                
+├── production                      <- environment directory
+│   ├── frontend-app                <- component directory
+│   |   └── terraform.tfvars        <- component configuration
+│   └── vpc
+│       └── terraform.tfvars
+├── staging                         <- environment directory
+│   ├── frontend-app                <- component directory
+│   |   └── terraform.tfvars        <- component configuration
+│   └── vpc
+│       └── terraform.tfvars
+└── terraform.tfvars                <- root configuration
+```
+
+The `live/` directory is the root of our Terragrunt configuration hierarchy.
+It contains a single `terraform.tfvars` file, which we will call
+the **root configuration**. The root configuration is optional, but allows 
+us to define general configuration that applies to all components in our infrastructure
+in a single place.
+
+The root contains a child directory for each environment in the infrastructure
+(e.g. production, staging), which we will call **environment directories**.
+The directory structure shown above contains two environment directories:
+`production` and `staging`.
+Environment directories do not normally contain any files. Instead, they are
+the location where Terragrunt commands such as `plan-all` and `apply-all`

```

Comment:
@dpetzel Terragrunt will traverse upwards beyond the directory you execute Terragrunt from to find the parent. I believe that it will traverse all the way up the tree until it finds a `terraform.tfvars` file, actually. Let me know if you have experienced otherwise and we'll sort it out in the example.